tests/conftest.py

```diff
@@ -209,3 +211,170 @@ def sample_bin() -> Bin:
             )
         ]
     )
+
+
+@pytest.fixture
+def sample_revision_two() -> Revision:
+    return Revision(
+        identifier=uuid.UUID("00000000-0000-0000-0000-000000000001"),
+        alternate_identifiers=["xyzzy:00000001"],
+        revision_number=2,
+        created_at=datetime(2025, 2, 11, 12, 0, 0, 0, tzinfo=UTC),
+        common_metadata={
+            "@schema": "urn:umich.edu:dor:schema:common",
+            "title": "Discussion also Republican owner hot already itself.",
+            "author": "Kimberly Garza",
+            "publication_date": "1989-04-16",
+            "subjects": [
+                "Liechtenstein",
+                "Vietnam",
+                "Something new"
```

Comment:
The only differences in `sample_revision_two` are the `revision_number`, `created_at`, and adding "Something new" to subjects in `common_metadata`. Any ideas on reducing duplication?

Reply:
Do you have access to `sample_revision` in this method? If you did, would it be bad form to have a method that duped `sample_revision` and update it?

Reply:
Thought occurred to me, I can try it to see how it feels.

Reply:
Not sure if it's bad form, but it was simple and much shorter, so I went for it: https://github.com/mlibrary/dor-py/pull/36/commits/a50d04f710871db609ecc63bd54162ec44969c43 I think it's okay since it's meant to be an evolution of `sample_revision` anyway.